ap/tasks/framework/base.py

```diff
@@ -118,6 +128,84 @@ def get_producer_kwargs(cls, task=None, **params):
         # implemented here only for simplified mro control
         return cls.get_array_function_kwargs(task=task, **params)
 
+    @classmethod
+    def find_config_objects(
+        cls,
+        names: Union[str, Sequence[str], Set[str]],
+        container: od.UniqueObject,
+        object_cls: type,
+        object_group: Optional[str] = None,
+        accept_patterns: bool = True,
+        deep: bool = False,
+        context: Optional[str] = None,
+    ) -> List[str]:
+        """
+        Returns all names of objects of type *object_cls* known to a *container* (e.g.
+        :py:class:`od.Analysis` or :py:class:`od.Config`) that match *names*. A name can also be a
+        pattern to match if *accept_patterns* is *True*, or, when given, the key of a mapping
+        *object_group* that matches group names to object names. When *deep* is *True* the lookup of
```

Comment:
Frankly, I had it like that before ^^. However, since the lookup also takes mappings called `object_group` in the aux data of the config, this became rather analysis-dependent and I had to move it out of "order".